modules/channel-web/src/views/lite/components/Composer.tsx

```diff
@@ -73,6 +72,7 @@ class Composer extends React.Component<ComposerProps> {
             value={this.props.message}
             onKeyPress={this.handleKeyPress}
             onKeyDown={this.handleKeyDown}
+            aria-label={this.props.lang === 'fr' ? 'Message à envoyer' : 'Message to send'}
```

Comment:
That fix was okay in the custom component, but translations are built-in in the module. Please check the en.json / fr.json file, and use the correct key (check the placeholder for example)